core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -517,6 +518,8 @@ protected[actions] trait PrimitiveActions {
   private def completeActivation(user: Identity, session: Session, response: ActivationResponse)(
     implicit transid: TransactionId): WhiskActivation = {
 
+    val context = UserContext(user)
```

Comment:
is the original request context needed here?

Reply:
Yes, the user information needs to be passed to the ActivationStore `store()` method. This allows ActivationStore SPIs to extract user information if need be,

Reply:
but what about the original request context (i.e. request headers, payload, etc)..  since there's is a write to the activation store (like in other places) i was thinking the request context would also be needed (like in other places)

Reply:
I don't have a use case for passing the request to the activation store here. Would rather keep the invocation path lean unless we really need the HTTP header in the store method.